### PR TITLE
Feature/change module structure

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Replaces #64

### CODE REVIEW

**Comment by smartie2076** [comment]
Thank you @Bachibouzouk ! @santiagoinfantinom can you check whether or not this includes all the changes you wanted to apply? Please request our feedback when ready.

**Comment by smartie2076** [comment]
Hey @santiagoinfantinom! What is the current status of this PR? Are you done with this?

**Comment by santiagoinfantinom** [comment]
@smartie2076 
I'm still working at it. 
I just can't get travis to work with the new structure. I created a setup.py but it's not working yet. I'll be focusing on doing this today.

**Comment by santiagoinfantinom** [comment]
@smartie2076 Now that I made some small changes regarding module importation, the tests seem to pass. I think it's now ready to be reviewed and merged.

**Comment by smartie2076** [comment]
Nice, I will check it out soon. Can you as a last thing rename the "code_folder" with "src" (as in source)? This seems to be the convention. You should not have to do much manually, it is just a "replace in path" when you are in the main directory!

File: A0_main_script.py [hunk 1]
```diff
@@ -14,31 +14,28 @@
...
-    from code_folder.D0_process_input import process_input_parameters, noise
-    from code_folder.E_blackouts_central_grid import central_grid
-    from code_folder.F_case_definitions import cases
-    from code_folder.G0_oemof_simulate import oemof_simulate
-    from code_folder.H0_multicriteria_analysis import multicriteria_analysis
+    from src.D0_process_input import process_input_parameters, noise
+    from src.E_blackouts_central_grid import central_grid
+    from src.F_case_definitions import cases
+    from src.G0_oemof_simulate import oemof_simulate
+    from src.H0_multicriteria_analysis import multicriteria_analysis
```
Comment (smartie2076) [inline A0_main_script.py:38]: I thought you wanted to figure out how this works without try/except, comparable to the MVS?

**Comment by smartie2076** [comment]
It was honestly a bit tough to really check the code step by step. I am going to trust you on this - as the simulation still runs through, chances are very high that everything still works ;)

**Comment by smartie2076** [comment]
After you have addressed about naming changes and the authorship, you can merge this branch into dev yourself.

**Pull request merged**